### ISSUE TITLE
Dragging image from other page into textarea doesn't add complete path

### ISSUE THREAD

**Issue by texnixe** [getkirby/kirby#1450]
**Describe the bug**
When dragging an image from a files section (files of another page than current page) the Panel adds just the filename, not the path to the filename, so that the image will not show up on frontend

**To Reproduce**
1. Create a files section, e.g in note.yml, set parent to `page.parent`.
2. Upload an image
3. Try to drag file from section into textarea

**Expected behavior**
The path to the file should be added

**Comment by bastianallgeier** [comment]
✅

**Comment by joernroeder** [comment]
the following line https://github.com/getkirby/kirby/commit/4367e6abb3eb065ec759ea952a91cda58b114d82#diff-4d30b3d1bc3cb53b71e98166837469c6R114 now introduces an error with my files section in `site.yml`
```
The section "logfiles" could not be loaded: Argument 1 passed to Kirby\Cms\Site::is() must be an instance of Kirby\Cms\Site, instance of Kirby\Cms\Page given, called in /var/www/localhost/htdocs/vendor/getkirby/kirby/config/sections/files.php on line 114 
```

```
sections:
  logfiles:
    type: files
    headline: Bisherige Export-Logs
    template: generator-log
    parent: site.find("generator")
```

The Error disappears when i remove `parent: site.find("generator")`

**Comment by distantnative** [comment]
@joernroeder just got fixed again today for 3.1: https://github.com/getkirby/kirby/commit/469ec12e4819524d9a5cb57bd3948e99d430a3a5

**Comment by joernroeder** [comment]
@distantnative ah perfect! thanks
